cmd/main.go

```diff
@@ -57,7 +73,14 @@ func main() {
 	})
 
 	e.GET("/players", func(c echo.Context) error {
-		return c.Render(http.StatusOK, "players", nil)
+		var players []*database.Player
+		result := db.Find(&players) // SELECT * FROM Players;
+		if result.Error != nil {
+			fmt.Errorf("Error loading players: %v", result.Error)
+			return c.HTML(http.StatusInternalServerError, "<p>Error fetching players.</p>")
```

Comment:
did this actually work?

Reply:
Yep - HTML at the page is just the content provided here, so just:
`<p>Error fetching players.</p>`

Reply:
Eventually we'll want to have htmx templates that display error messages inline, or something nicer than this, but this does technically work.